### PR TITLE
Removed unneeded encoding of credentials download

### PR DESCRIPTION
**Pull request by jinapurapu**
Fixes https://github.com/minio/console/issues/2031

### CODE REVIEW

**Review by bexsoft — APPROVED** [review]
LGTM

File: portal-ui/src/screens/Console/Common/CredentialsPrompt/CredentialsPrompt.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const download = (filename: string, text: string) => {
   let element = document.createElement("a");
   element.setAttribute(
     "href",
-    "data:text/plain;charset=utf-8," + encodeURLString(text)
+    "data:text/plain;charset=utf-8," + text
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Common/CredentialsPrompt/CredentialsPrompt.tsx:80]: Please remove import for this:

https://github.com/minio/console/pull/2032/files#diff-9989d3345835ccb52d5ca352ea42ce0c6d4491b0aaca18f522b348da477fb08aR30

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]
:+1:

**Pull request merged**